_posts/2017/09/2017-09-11-lesson-infrastructure-subcommittee.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Lesson Infrastructure Subcommittee 2017 September meeting"
+date: 2017-09-12
+time: "10:00:00"
+category: ["Community", "Lesson Infrastructure Subcommittee"]
+---
+
+On 5 September 2017 at 14:00UTC+0,
+the Lesson Infrastructure Subcommittee had their 2017 September meeting.
+This post will cover the topics discussed and their resolutions.
+
+## Software Carpentry and Data Carpentry merge
+
+With the [upcoming merge](https://software-carpentry.org/blog/2017/09/merger.html),
+this subcommittee needs to start thinking about streamline the thinking process and disconect from the organisations that maintain the lessons.
+The responsibilities for this subcommittee will stay unchanged:
+
+- maintain [lesson template](https://github.com/swcarpentry/styles)
+- maintain [lesson documentation](https://github.com/swcarpentry/lesson-example)
+- maintain [workshop template](https://github.com/swcarpentry/workshop-template)
+- overview of what features the lessons will continue to have
+- stay in contact with maintainers of lesson
+- stay in contact with staffs
+
+[Lesson template](https://github.com/swcarpentry/styles),
+[lesson documentation](https://github.com/swcarpentry/lesson-example)
+and
+[workshop template](https://github.com/swcarpentry/workshop-template)
+**will have a new home in 2018**.
+If you have questions or want to help with this migration,
+[Christina Koch](mailto:koch.christinal@gmail.com) is the person you should contact.
+
+During the migration,
+we will solve the divergences between
+[Software Carpentry workshop template](https://github.com/swcarpentry/workshop-template)
+and
+[Data Carpentry workshop template](https://github.com/datacarpentry/workshop-template).
+If you have questions or want to help with this migration,
+[Maneesha Sane](mailto:maneesha@software-carpentry.org) is the person you should contact.
+
+## Keyboard key visual look
+
+To improve the reader experience of the lesson,
+we will make a visual diference of keyboard keys that need to be pressed by the reader.
```

Comment:
we will make the keyboard keys that need to be pressed by the learner, look different from the other components of the text, so they are highlighted more effectively.